From b50459bde21f78f1bc3594efcc2acd436dac11e6 Mon Sep 17 00:00:00 2001
From: mayank <mbabariya2000@gmail.com>
Date: Sun, 16 Mar 2025 03:38:06 +0000
Subject: [PATCH] Hello World from Mayank

---
 testsuites/samples/hello/init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuites/samples/hello/init.c b/testsuites/samples/hello/init.c
index 83f6342ab3..95ed506a45 100644
--- a/testsuites/samples/hello/init.c
+++ b/testsuites/samples/hello/init.c
@@ -33,7 +33,7 @@
 #include <rtems.h>
 #include <tmacros.h>
 
-const char rtems_test_name[] = "HELLO WORLD";
+const char rtems_test_name[] = "Hello World from Mayank";
 
 static rtems_task Init(
   rtems_task_argument ignored
-- 
2.43.0

